src/common/options.cc

```diff
@@ -5226,6 +5226,14 @@ static std::vector<Option> get_rbd_options() {
     Option("rbd_journal_max_concurrent_object_sets", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(0)
     .set_description("maximum number of object sets a journal client can be behind before it is automatically unregistered"),
+
+    Option("rbd_image_qos_iops_read_limit", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
Nit: ```_image_``` is unnecessary since all ```rbd_``` options are for an image. Might also be good to name these ```rbd_read_iops_max``` and ```rbd_write_iops_max```.